### ISSUE TITLE
DHT: Address family not supported by protocol

### ISSUE THREAD

**Issue by leonghui** [aria2/aria2#793]
First issue here, thanks for this awesome and invaluable project.

Reproduced on version 1.28.0, running on LEDE nightly

The error below appears in the log regardless of the value of enable-dht6 (tried true/false/unspecified).

Magnet links will not start until I set disable-ipv6=true

```
[INFO] [Context.cc:183] Linux 4.4.32 #0 Tue Nov 29 11:19:22 2016 mips
[INFO] [Context.cc:184] zlib/1.2.8 libxml2/2.9.4 OpenSSL/1.0.2j
...
[INFO] [DHTMessageDispatcherImpl.cc:88] Failed to send message: dht query ping TransactionID=d57eef21 Remote:2001:41d0:c:5ac:x::x(6881), id=7a7ec70a0489f78b1eb6be25dc23153bf7bff4c6, v=A2%00%03,
Exception: [SocketCore.cc:1217] errorCode=1 Failed to send data, cause: Address family not supported by protocol
```

**Comment by tatsuhiro-t** [comment]
If you are using --dht-entry-point option or torrent file which includes nodes data, then #794 fixes this issue.

**Comment by leonghui** [comment]
Will retest with v1.30.0. Thanks!

**Comment by leonghui** [comment]
Issued resolved with v1.30.0. Thanks again!
